.github/workflows/release.yml

```
@@ -347,117 +344,147 @@
           dart pub global activate melos 6.3.2
           melos bootstrap
~      
~      # Get version from pubspec.yaml
       - name: Get version from pubspec.yaml
         id: get_version
         run: |
-          VERSION=$(grep '^version:' apps/flipper/pubspec.yaml | cut -d ' ' -f 2 | tr -d '\r')
+          VERSION=$(grep '^version:' apps/${{ matrix.app }}/pubspec.yaml | cut -d ' ' -f 2 | tr -d '\r')
           echo "version=$VERSION" >> $GITHUB_OUTPUT
           echo "Version: $VERSION"
         
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)